django_project/base/migrations/0019_project_project_repository_url.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('base', '0018_auto_20180220_0339'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='project',
+            name='project_repository_url',
+            field=models.URLField(help_text="A repository URL for this project. For instance a path to the project's GitHub repository.", null=True, blank=True),
```

Comment:
PEP8

Reply:
This is a migration file. They are autogenerated by django so excluded from the PEP8 test in Travis and makefile.